modules/javafx.fxml/src/main/java/javafx/fxml/FXMLLoader.java

```diff
@@ -1497,6 +1490,11 @@ public void processCharacters() throws IOException {
 
             updateValue(text.trim());
         }
+
+        @Override
```

Comment:
why removing it from the base class, just to add it back in multiple places?  is this really necessary?

Reply:
The base method was empty, and it throws `IOException` which triggers a warning.  By making it `abstract` the warning is gone (I could also have documented it).  Since this is an internal private class, it is fine to make it `abstract` instead (the class itself was already `abstract`).  This also means there is no need to call the super implementation since it is now abstract and known to be empty.